packages/components/src/components/hds/tabs/index.ts

```diff
@@ -35,13 +35,13 @@ export interface HdsTabsSignature {
 }
 
 export default class HdsTabs extends Component<HdsTabsSignature> {
-  @tracked tabNodes: HTMLButtonElement[] = [];
-  @tracked tabIds: HdsTabsTabIds = [];
-  @tracked panelNodes: HTMLElement[] = [];
-  @tracked panelIds: HdsTabsPanelIds = [];
-  @tracked _selectedTabIndex;
-  @tracked selectedTabId?: string;
-  @tracked isControlled: boolean;
+  @tracked private _tabNodes: HTMLButtonElement[] = [];
+  @tracked private _tabIds: HdsTabsTabIds = [];
+  @tracked private _panelNodes: HTMLElement[] = [];
+  @tracked private _panelIds: HdsTabsPanelIds = [];
+  @tracked private _selectedTabIndex;
```

Comment:
@alex-ju wasn't sure what to do here. This is a prop that mirrors the public argument, but currently it uses the private notation. Should we update this one to use the public notation? Not sure if in the past it was notated as private for some reason.